### ISSUE TITLE
Joint Training of Candidate Extraction and Answer Selection for Reading Comprehension

### ISSUE THREAD

**Issue by Toshiki-Sasaki** [Toshiki-Sasaki/papers#66]
## Abstract 
Reading Comprehensionの分野で洗練されたneural-basedの技術が開発されてきた．
ほとんどのアプローチが他の答えの候補との関係を無視し，答えを独立様式に合わせている．
この問題は，複数のpassageからの候補が一つの質問に答えるのに組み込まれるopen-domainの問題でさらに問題となる．
この論文では，我々はRCを抽出-選択の2段階処理として形式化する．
我々じゃ最初にpassageから答えの候補を抽出し，全ての候補の情報を組み込むことにより最後の答えを選択します．
さらに，我々は潜在変数として候補の抽出を考慮し，二段階処理を強化学習と結合し訓練します．
結果，我々のアプローチは，2つのopen-domainのデータセットでSOTAのパフォーマンスをかなり改善しました．
追加の分析では，モデル構造の効果を示しました．
特に全ての候補の情報結合と，抽出-選択処理の結合訓練についてです．

## ポイント
タスク：
データセット：
特徴1 : 
特徴2 : 
特徴3 : 

### 実験
word embedding : 
tokenizer : 
BiLSTM : 
hidden-dim : 
mini-batch size : 
Optimizer : 
learning rate : 
dropout rate :   

### 結果
xxx
